frontend/src/Store/Actions/seriesActions.js

```diff
@@ -377,8 +379,56 @@ export const actionHandlers = handleThunks({
           });
         });
     }, MONITOR_TIMEOUT);
-  }
+  },
+
+  [UPDATE_SERIES_MONITOR]: function(getState, payload, dispatch) {
+    const {
+      id,
+      monitored,
+      monitor
+    } = payload;
+
+    const series = [];
+    const seriesToUpdate = { id };
+    if (payload.hasOwnProperty('monitored')) {
+      seriesToUpdate.monitored = monitored;
+    }
+    series.push(seriesToUpdate);
 
+    dispatch(set({
+      section,
+      isSaving: true
+    }));
+
+    const promise = createAjaxRequest({
+      url: '/seasonPass',
+      method: 'POST',
+      data: JSON.stringify({
+        series,
+        monitoringOptions: { monitor }
+      }),
+      dataType: 'json'
+    }).request;
+
+    promise.done((data) => {
+      dispatch(fetchSeries());
+
+      dispatch(set({
+        section,
+        isSaving: false,
+        saveError: null
+      }));
+    });
+
+    promise.fail((xhr) => {
+      dispatch(set({
+        section,
+        isSaving: false,
+        saveError: xhr
+      }));
+    });
+    location.reload();
```

Comment:
Left over from debugging? Or a way to update the page? If the saving the modal should trigger an episode fetch after the saving finishes.

Reply:
It was a way to both update the page and dismiss the modal (couldn't work out why the modal wasn't closing), I will look at this bit again

Reply:
Typically that's done when `isSaving` completes in a `componentDidUpdate` function (at least in our pre-React Hook world).

Reply:
I have removed the location.reload() and got the modal to close, it doesn't however update the page still - I had a look at the toggle monitor/unmonitor for an episode and this also doesn't update the page so I believe this is in keeping with other similar features of Sonarr. 

Ideally I want it to update the page so it's more obvious that it has done what you have asked it to but unsure how to make that work as fetchEpisodes broke the page!

Reply:
> I had a look at the toggle monitor/unmonitor for an episode and this also doesn't update the page so I believe this is in keeping with other similar features of Sonarr.

That's because there isn't anything to update. Toggling monitored at the series level doesn't change episodes. Toggling it at the season level does and the episode are updated, though that's locally.

> Ideally I want it to update the page so it's more obvious that it has done what you have asked it to but unsure how to make that work as fetchEpisodes broke the page!

Yeah it definitely needs to be updated. Take a look how season pass works, after the changes are applied the affected series are fetched, which is part of it, since both the series and the episodes in the series need to be fetched.

Reply:
This is now working